### DISCUSSION TITLE
Purchased unknowingly

### DISCUSSION THREAD

**Discussion by colinschlecht** [aclap-dev/video-downloadhelper#924] [License & Purchase requests]
### What's your issue?

I just see a charge on my acct from 5/30 for $18. I don’t use this app and didn’t intentionally purchase this?

### The 10 first digits (only 10) of your license number or email address

Schlechtcolin@yahoo.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
hi,
sorry I did not find your purchase, please send us details at licenses@downloadhelper.net

**Comment by boe308** [comment]
hi sorry i got the chrome version by mistake i wanted the edge version i didnt know its diffrent ones for each browser

**Reply by jcfrog** [reply]
hi,
please send us an email at licenses@downloadhelper.net
